src/stories/ThreeOneOne.stories.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { checkA11y } from '@storybook/addon-a11y';
+import { IntlProvider } from 'react-intl';
+
+
+import ThreeOneOne from 'js/page_sections/ThreeOneOne';
+import jsonFileData from '__tmpdata/pages';
```

Comment:
should we keep static data for stories in one place? Maybe we can copy this file to stories/static_data/ so it can work for both until we get 311 data in joplin for the static app.